tests/migration/online_migration/yast2_migration.pm

```diff
@@ -208,6 +209,10 @@ sub run {
     assert_screen 'yast2-migration-target-list-selected', 60;
     send_key_until_needlematch 'migration-target-' . get_var("VERSION"), 'down', 20, 3;
     send_key "alt-n";
+    assert_screen 'import-untrusted-gpg-key', 60 if (is_leap_migration);
```

Comment:
It's better to change the if sentence as below to avoid run later no-need judgement sentence when not matched for the is_leap_migration.
if (is_leap_migration) {
    assert_screen 'import-untrusted-gpg-key', 60;
    if (match_has_tag('import-untrusted-gpg-key')) {
        send_key 'alt-t';
    }
}

Reply:
changed!